test/components/remote_retro_test.js

```diff
@@ -32,4 +39,28 @@ describe("RemoteRetro component", () => {
       expect(hotjarSpy).calledWith("trigger", CLOSED)
     })
   })
+
+  describe("when component updates with a new facilitator", () => {
+    let newFacilitator
+    beforeEach(() => {
+      newFacilitator = {
+        ...stubUser,
+        is_facilitator: true,
+      }
+    })
+
+    it("calls the new facilitator action", () => {
+      const wrapper = shallow(<RemoteRetro {...defaultProps} />)
+      wrapper.setProps({ user: newFacilitator })
+      expect(defaultProps.actions.newFacilitator.calledWith(defaultProps.retro))
+    })
+  })
+
+  describe("when component updates new stage", () => {
+    it("does not the new facilitator action", () => {
+      const wrapper = shallow(<RemoteRetro {...defaultProps} />)
+      wrapper.setProps({ stage: CLOSED })
+      expect(defaultProps.actions.newFacilitator.notCalled)
```

Comment:
will this test always pass? I think we need an assertion attached to the `expect()`, but i could be wrong.

Reply:
we do, an oversight on my part